### PR TITLE
Add post rss feed

### PR DESCRIPTION
**Pull request by sarahquigley**
- Add RSS and Atom Feeds for Posts.
- Begin using Django's 'sites' framework for info about current site.

### CODE REVIEW

**Comment by sarahquigley** [comment]
@mstewart Take a quick look at this PR and let me know what you think.

File: django_basic_blog/settings.py [hunk 1]
```diff
@@ -105,6 +106,11 @@
 
 USE_TZ = True
 
+# Django sites settings
+# https://docs.djangoproject.com/en/1.8/ref/contrib/sites/
+
+SITE_ID = 1
```
Comment (mstewart) [inline django_basic_blog/settings.py]: Drop this?

**Comment by mstewart** [comment]
:+1:

**Comment by sarahquigley** [comment]
Thanks @mstewart :heart:

**Pull request merged**